docs/Encryption.md

```diff
@@ -0,0 +1,76 @@
+S3Proxy
+
+# Encryption 
```

Comment:
Are there any other precedents that we can follow for S3 client-side encryption, e.g., from AWS SDK or some other project?

Reply:
I looked in various implementations and none of them satisfice me as it comes always with some limitations that needs to taken in consideration like on AWS SDK you will need to upload the client side encrypted parts in the right order etc.